docs/bookmarks.md

```diff
@@ -0,0 +1,221 @@
+  commit `jj show <bookmark name>@<remote name>` would show. This notion is
+  completely analogous to Git's "remote-tracking bookmarks".
+- A **tracked (remote) bookmark** is defined above. You can make a remote bookmark
+  tracked with the [`jj bookmark track` command](#manually-tracking-a-bookmark), for
+  example.
+- A **tracking (local) bookmark** is the local bookmark that `jj` tries to keep in
+  sync with the tracked remote bookmark. For example, after `jj bookmark track
+  mybookmark@origin`, there will be a local bookmark `mybookmark` that's tracking the
+  remote `mybookmark@origin` bookmark. A local bookmark can track a bookmark of the same
+  name on 0 or more remotes.
+
+The notion of tracked bookmarks serves a similar function to the Git notion of an
+"upstream branch". Unlike Git, a single local bookmark can be tracking remote
+bookmarks on multiple remotes, and the names of the local and remote bookmarks
+must match.
+
+### Manually tracking a bookmark
+
+To track a bookmark permanently use `jj bookmark track <bookmark name>@<remote name>`. 
+It will now be imported as a local bookmark until you untrack it or it is deleted
+on the remote. 
+
+Example:
+
+```sh
+$ # List all available bookmarks, as we want our colleague's bookmark.
+$ jj bookmark list --all
+$ # Find the bookmark.
+$ # [...]
+$ # Actually track the bookmark.
+$ jj bookmark track <bookmark name>@<remote name> # Example: jj bookmark track my-feature@origin
+$ # From this point on, <bookmark name> will be imported when fetching from <remote name>.
+$ jj git fetch --remote <remote name>
+$ # A local bookmark <bookmark name> should have been created or updated while fetching.
+$ jj new <bookmark name> # Do some local testing, etc.
+```
+
+### Untracking a bookmark
+
+To stop following a remote bookmark, you can `jj bookmark untrack` it. After that,
+subsequent fetches of that remote will no longer move the local bookmark to match
+the position of the remote bookmark.
+
+Example: 
+
+```sh
+$ # List all local and remote bookmarks.
+$ jj bookmark list --all
+$ # Find the bookmark we no longer want to track.
+$ # [...]
+# # Actually untrack it.
+$ jj bookmark untrack <bookmark name>@<remote name> # Example: jj bookmark untrack stuff@origin
+$ # From this point on, this remote bookmark won't be imported anymore.
+$ # The local bookmark (e.g. stuff) is unaffected. It may or may not still
+$ # be tracking bookmarks on other remotes (e.g. stuff@upstream).
+```
+
+### Listing tracked bookmarks
+
+To list tracked bookmarks, you can `jj bookmark list --tracked` or `jj bookmark list -t`.
+This command omits local Git-tracking bookmarks by default.
+
+You can see if a specific bookmark is tracked with `jj bookmark list --tracked <bookmark name>`.
+
+
+### Automatic tracking of bookmarks & `git.auto-local-bookmark` option
+
+There are two situations where `jj` tracks bookmarks automatically. `jj git
+clone` automatically sets up the default remote bookmark (e.g. `main@origin`) as
```

Comment:
Assuming we're making this change, "default remote branch" here and "newly created branch on the remote" below.